packages/components/Atoms/Icon/stories.js

```diff
@@ -112,7 +115,7 @@ storiesOf('Atoms/Icon', module)
           'xls',
           'xlsm',
           'xlsx',
-          'youtube',
+          'youtube_channel',
```

Comment:
Hey @rubyllamadora rename this may will cause issue as I remembered one logic use BE string value to pick youtube icon, rename icon may cause the logic fail.